code/modules/events/crystal_event.dm

```diff
@@ -0,0 +1,412 @@
+
+/*
+This section is for the crystal monsters variations
+*/
+
+/mob/living/simple_animal/hostile/crystal_monster
+	name = "crystal monster"
+	desc = "A monster made of crystals similar to the Supermatter ones."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "crystal_minion"
+	icon_living = "crystal_minion"
+	icon_dead = "crystal_minion"
+	gender = NEUTER
+	mob_biotypes = MOB_UNDEAD|MOB_HUMANOID
+	turns_per_move = 5
+	speak_emote = list("resonates")
+	emote_see = list("resonates")
+	a_intent = INTENT_HARM
+	maxHealth = 25
+	health = 25
+	speed = 1.2
+	harm_intent_damage = 2.5
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	minbodytemp = 0
+	maxbodytemp = 1500
+	healable = 0 //they're crystals how would bruise packs help them??
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	attack_sound = 'sound/effects/supermatter.ogg'
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 10
+	robust_searching = 1
+	stat_attack = UNCONSCIOUS
+	faction = list("crystal")
+	see_in_dark = 8
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	deathmessage = "collapses into dust!"
+	del_on_death = 1
+	footstep_type = FOOTSTEP_MOB_SHOE
+
+/mob/living/simple_animal/hostile/crystal_monster/minion
+	name = "crystal minion"
+	desc = "A monster made of crystals similar to the Supermatter ones."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "crystal_minion"
+	icon_living = "crystal_minion"
+	icon_dead = "crystal_minion"
+	maxHealth = 25
+	health = 25
+	speed = 1.2
+	harm_intent_damage = 2.5
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+
+/mob/living/simple_animal/hostile/crystal_monster/thug
+	name = "crystal thug"
+	desc = "A monster made of crystals similar to the Supermatter ones."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "crystal_thug"
+	icon_living = "crystal_thug"
+	icon_dead = "crystal_thug"
+	maxHealth = 35
+	health = 35
+	speed = 1
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 7
+/mob/living/simple_animal/hostile/crystal_monster/recruit
+	name = "crystal recruit"
+	desc = "A monster made of crystals similar to the Supermatter ones."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "crystal_recruit"
+	icon_living = "crystal_recruit"
+	icon_dead = "crystal_recruit"
+	maxHealth = 45
+	health = 45
+	speed = 1
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+/mob/living/simple_animal/hostile/crystal_monster/killer
+	name = "crystal killer"
+	desc = "A monster made of crystals similar to the Supermatter ones."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "crystal_killer"
+	icon_living = "crystal_killer"
+	icon_dead = "crystal_killer"
+	maxHealth = 60
+	health = 60
+	speed = 0.9
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+/mob/living/simple_animal/hostile/crystal_monster/boss
```

Comment:
these crystal dudes are kind of pathetically weak

also, why do crystal minions take half as much damage from barehanded punches than the other kinds of crystal monsters do?

Reply:
i just eyeballed the numbers, i have no clue how powerful is powerful, what would be a good reference point to do this?

Reply:
i checked the spiders, made the crystal monster around those values